README.md

```diff
@@ -43,4 +43,16 @@ jobs:
 | `run-tests` | Enable/Disable running tests.  | Optional | `"false"` |
 | `branch` | The default flatpak branch.  | Optional | `master` |
 
-The Docker Image used can be found [here](./docker/Dockerfile).
+## Docker Image
+
+The Docker image used for the action consists of 2 parts: The base image, based on Fedora and which can be found
+[here](docker/Dockerfile), and the specific image of the runtime you choose, which is generated through
+[this](.github/workflows/docker.yml) GitHub Actions workflow.
+
+You can specify the specific runtime you need to use through the image tags:
+
+| Runtime | Tag | Example |
+| ------- | --- | ------- |
+| freedesktop.org 20.08 | `fdo-20-08` | `image: bilelmoussaoui/flatpak-github-actions:fdo-20-08` |
```

Comment:
First: it's not freedesktop.org at all just write it as `Freedesktop SDK` (see here https://freedesktop-sdk.io/). Also I would prefer if we split the runtime into two columns, one called Runtime & the other Version

Reply:
Done